parquet/sandboxes/heka/output/s3_parquet.lua

```diff
@@ -112,6 +114,8 @@ require "parquet"
 local load_schema = require "lpeg.parquet".load_parquet_schema
 require "string"
 require "table"
+local date          = require "os".date
+local floor         = require "math".floor
```

Comment:
I'm using it on line 250, and without it os.date() will throw the error: `bad argument #2 to 'date' (number has no integer representation)`